### PR TITLE
proposer set update logic for governable.sol

### PR DESCRIPTION
**Pull request by saiakilesh**
# Questions
Who should be able to set the session length multiplier?

### CODE REVIEW

File: contracts/utils/Governable.sol [hunk 1]
```diff
@@ -98,5 +120,60 @@ contract Governable {
...
+
+        proposerSetRoot = _proposerSetRoot;
+        averageSessionLengthInMillisecs = _averageSessionLengthInMillisecs;
+        numOfProposers = _numOfProposers;
+        proposerSetUpdateNonce = _proposerSetUpdateNonce;
+        currentVotingPeriod++;
+    }
+
+    function voteInFavorForceSetGovernor(Vote memory vote) external {
+
```
Comment (drewstone) [inline contracts/utils/Governable.sol:143]: Extra line

File: test/governance/governable.test.ts [hunk 2]
```diff
@@ -4,15 +4,16 @@
...
  import BN from 'bn.js';
- import { toHex } from '../../packages/utils/src';
+ import { toFixedHex, toHex } from '../../packages/utils/src';
  import EC from 'elliptic';
  const ec = new EC.ec('secp256k1');
  const TruffleAssert = require('truffle-assertions');
  
  // Convenience wrapper classes for contract classes
  import { Governable__factory } from '../../typechain';
+import { BlockForkEvent } from '@ethersproject/contracts/node_modules/@ethersproject/abstract-provider';
```
Comment (drewstone) [inline test/governance/governable.test.ts:16]: Is this needed?

**Review by drewstone — APPROVED** [review]
LGTM pending small changes.

File: test/governance/governable.test.ts (hunk 2 above)
Comment (saiakilesh) [reply test/governance/governable.test.ts]: fixed

**Pull request merged**